com.stencyl.Game.json

```diff
@@ -0,0 +1,153 @@
+{
+    "app-id": "com.stencyl.Game",
+    "runtime": "com.endlessm.Sdk",
+    "runtime-version": "eos3.1",
+    "sdk": "com.endlessm.Sdk",
+    "command": "stencyl",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--share=network",
+        "--device=all",
+        "--filesystem=host",
+        "--allow=multiarch",
+        "--env=JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64",
+        "--extra-data=Stencyl-64-full.tar.gz:c27d00dc83b5f443c35d785a0d22340f35c1cdb289c8cfe30417963475d4af29:207725021::http://mario.stencyl.net/public/Stencyl-64-full.tar.gz"
+    ],
+    "modules": [
+        {
+          "name": "libxext",
+          "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://archive.ubuntu.com/ubuntu/pool/main/libx/libxext/libxext_1.3.0.orig.tar.gz",
+                    "sha256": "471a4343940f62985274ee411cd1f7d04cb8090b1e8fd15513d5c064851752fe"
+                }
+          ]
+        },
+        {
+          "name": "libgc",
+          "no-autogen": true,
+          "sources": [
+                {
+                    "type": "file",
+                    "url": "http://de.archive.ubuntu.com/ubuntu/pool/main/libg/libgc/libgc1c2_7.4.2-8_amd64.deb",
+                    "sha256": "f45cf311444ecf1c3170412c5ad4ae3e6ba8d66cc7c3e59ff6177d1c29d2c651",
+                    "dest-filename": "libgc.deb"
+                },
+                {
+                    "type": "file",
+                    "path": "deb-Makefile",
+                    "dest-filename": "Makefile"
+                }
+          ]
+        },
+        {
+            "name": "java&resources",
```

Comment:
So, Lili told me that the modules would need a Makefile anyway, so she merged them into one. Not sure if I like the explanation, but fair enough for now.